packages/react-client/src/hooks/internal.ts

```diff
@@ -1 +1 @@
-export { useFishjamClient_DO_NOT_USE } from "./useFishjamClient";
+export { useFishjamClient_DO_NOT_USE } from "./useFishjamClient_DO_NOT_USE";
```

Comment:
Why we need this file? 🤔

Reply:
For the Videoroom to continue working. The Videoroom logic requires events and methods from the `ts-client`. With current API it's impossible to e.g. request different simulcast layer

Reply:
i think we should extend our api to cover videoroom needs in the future. lets do it like that rn but leave a followup ticket

Reply:
I totally agree; the videoroom needs to be simplified, and our new API should be flexible enough to support that.
https://linear.app/swmansion/issue/FCE-560/simplify-the-videroom-codebase
https://linear.app/swmansion/issue/FCE-561/remove-usefishjamclient-do-not-use